lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -68,12 +68,25 @@ def generate
           # that use Swift. Setting this for the embedded target would
           # cause an App Store rejection because frameworks cannot be embedded
           # in embedded targets.
-          if !target.requires_host_target? && pod_targets.any?(&:uses_swift?)
-            config['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'YES'
-            config['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = 'YES'
+
+	  # Extract xcodeversion number
+	  @xcodeversion = 'xcodebuild -version | cut -f 2 -d ' ' | head -n 1`
```

Comment:
doing this sort of check doesn't really work, as the user can be building with a different version of Xcode than is reported when just running `xcodebuild`